docs/getting-started/initilize_ilab.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Initializing InstructLab
+description: Steps to initilize InsutrctLab on a local machine
+logo: images/ilab_dog.png
+---
+
+# 🏗️ Initialize `ilab`
+
+1) Initialize `ilab` by running the following command:
+
+```shell
+ilab config init
+```
+
+*Example output*
+
+```shell
+Welcome to InstructLab CLI. This guide will help you set up your environment.
+Please provide the following values to initiate the environment [press Enter for defaults]:
+Path to taxonomy repo [taxonomy]: <ENTER>
+```
+
+2) When prompted by the interface, press **Enter** to add a new default `config.yaml` file.
+
+3) When prompted, clone the `https://github.com/instructlab/taxonomy.git` repository into the current directory by typing **y**.
+
+   **Optional**: If you want to point to an existing local clone of the `taxonomy` repository, you can pass the path interactively or alternatively with the `--taxonomy-path` flag.
+
+   *Example output after initializing `ilab`*
+
+   ```shell
+   (venv) $ ilab config init
+   Welcome to InstructLab CLI. This guide will help you set up your environment.
+   Please provide the following values to initiate the environment [press Enter for defaults]:
+   Path to taxonomy repo [taxonomy]: <ENTER>
+   `taxonomy` seems to not exists or is empty. Should I clone https://github.com/instructlab/taxonomy.git for you? [y/N]: y
+   Cloning https://github.com/instructlab/taxonomy.git...
+   ```
+
+   `ilab` will use the default configuration file unless otherwise specified. You can override this behavior with the `--config` parameter for any `ilab` command.
+
+4) When prompted, provide the path to your default model. Otherwise, the default of a quantized [Merlinite](https://huggingface.co/instructlab/merlinite-7b-lab-GGUF) model will be used - you can download this model with `ilab model download` (see below).
+
+   ```shell
+   (venv) $ ilab config init
+   Welcome to InstructLab CLI. This guide will help you set up your environment.
+   Please provide the following values to initiate the environment [press Enter for defaults]:
+   Path to taxonomy repo [taxonomy]: <ENTER>
+   `taxonomy` seems to not exists or is empty. Should I clone https://github.com/instructlab/taxonomy.git for you? [y/N]: y
+   Cloning https://github.com/instructlab/taxonomy.git...
+   Path to your model [/Users/USERNAME/Library/Caches/instructlab/models/merlinite-7b-lab-Q4_K_M.gguf]: <ENTER>
```

Comment:
This is also assuming Mac, the `/Users/` is specific to the Mac, installation, maybe we should make that clear here.

Reply:
Good point! I'll clarify that here!